src/views/EditProfile.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div id="page">
+        <img width="210" height="70" src="http://placehold.jp/65/ffffff/666666/210x70.png?text=ProLab" alt="prolag">
+        <!-- <img alt="Prolab Logo" src="../assets/logo.png"> -->
+        <h1>プロフィール編集ページ</h1>
+        <b id="uname">user: prolab_user</b>
+        <form>
+            <input type="checkbox" name="profile_publish" id="checkbox" value="enabled">
+            プロフィールを公開する<br>
+            <b>表示名</b><br>
```

Comment:
(WIP なのにコメントしてごめんよ……)
HTML5 的に、b 要素だと違う意味になっちゃうので、使用は避けよう
https://developer.mozilla.org/ja/docs/Web/HTML/Element/b

ここでは label を使うのがいいっすね
https://developer.mozilla.org/ja/docs/Web/HTML/Element/label
```suggestion
            <label for="display_name">表示名</label>
            <input type="text" id="display_name" name="display_name" value="user">
```

Reply:
HTML5だと太字は&lt;strong&rt;とかCSSでboldするのがいいんですね。
ありがとうございます。明日修正してまた上げます